### DISCUSSION TITLE
Does lmdeploy support llama3 deployment?

### DISCUSSION THREAD

**Discussion by Lisaiplus** [InternLM/lmdeploy#1475] [Q&A]
If deployment is supported, why are the responses nonsensical? I'm using the Llama-3-8B-Instruct model, and it works fine when I test it with streamlit.
<img width="1178" alt="image" src="https://github.com/InternLM/lmdeploy/assets/100062430/be78f986-282d-4477-9de8-ca09069e4109">

**Comment by lvhan028** [comment]
It is supported in latest main branch but not released yet.
